src/Glpi/Controller/AbstractController.php

```diff
@@ -75,4 +75,18 @@ final protected function render(
         $response->setContent($content);
         return $response;
     }
+
+    /**
+     * Common Twig templates parameters
+     *
+     * @return array
+     */
+    protected function commonTwigParameters(): array
+    {
+        /** @var array $CFG_GLPI */
+        global $CFG_GLPI;
+        return [
```

Comment:
This fix missing lang on redirection page

Reply:
By `commonTwigParameters` do you mean parameters that should always be available ?

In this case, maybe you should call this new method directly from the `render()` method above.

Reply:
Well... I'm not entirely sure it must always be available. I first add this one in the `IndexController` class (since this is only used there for now); but making it available at higher level did seem a better idea.

Reply:
Yes, I think it depend on the name and what you do with it.

If its named `commonTwigParameters` and needed in multiple places, it probably need to be called directly in the render method.

If it is specific to the login controller, then it should handle it itself without change to the abstract parent.

Reply:
I removed the method